src/effects/native/bessel4lvmixeqeffect.cpp

```diff
@@ -10,12 +10,13 @@ QString Bessel4LVMixEQEffect::getId() {
 EffectManifest Bessel4LVMixEQEffect::getManifest() {
     EffectManifest manifest;
     manifest.setId(getId());
-    manifest.setName(QObject::tr("Bessel4 LV-Mix EQ"));
-    manifest.setShortName(QObject::tr("Bessel4 EQ"));
+    manifest.setName(QObject::tr("Bessel4 LV-Mix ISO"));
+    manifest.setShortName(QObject::tr("Bessel4 ISO"));
     manifest.setAuthor("The Mixxx Team");
     manifest.setVersion("1.0");
     manifest.setDescription(QObject::tr(
-        "A Bessel 4th order filter equalizer with Lipshitz and Vanderkooy mix (bit perfect unity, roll-off -24 db/Oct). "
+        "A Bessel 4th order filter isolator with Lipshitz and Vanderkooy mix (bit perfect unity, roll-off -24 db/Oct).") +
```

Comment:
Same, it feels awkward -- I'm also fine with that but we should be consistent :).